### DISCUSSION TITLE
404 on all standard laravel routes after updating to v3.2.31

### DISCUSSION THREAD

**Discussion by CodeMooseVTG** [statamic/cms#5122] [Troubleshooting]
Statamic Pro v3.2.31
Laravel v8.80.0

Hi there! I recently updated all the packages in my install - now any routes defined outside Statamic are returning 404. I have one route in my `web.php` that is used to pull in some external data with a default Laravel controller:

```
use Illuminate\Support\Facades\Route;
use App\Http\Controllers\SearchController;

Route::group(['prefix' => 'public-api'], function() {
	Route::post( 'search', [SearchController::class, 'getSearchResults'] )->name( 'search' );
});
```

To be clear, this route was working perfectly before the update. I strongly suspect this is a Statamic issue, as when I disable routing in `config/statamic/routes.php` everything works as expected. I also set up a simple "Hello World" route in a new controller to make sure it wasn't a typo or something about the search controller implementation - the simplest possible controller method still returns 404, and works when Statamic routing is disabled.

Has anyone else run into this? I can't find any other threads or documentation about similar issues - is there some new mechanism I need to engage to allow normal routes again?

**Comment by jasonvarga** [comment]
What version did you upgrade from?

**Reply by CodeMooseVTG** [reply]
Last version we were on was v3.2.4

**Reply by jasonvarga** [reply]
If you add `Route::get('foo', function () { return 'foo'; });` to your `routes/web.php` and visit `/foo`, what do you see? Still a 404?

**Reply by CodeMooseVTG** [reply]
Indeed - still get a 404, although strangely it still 404s when the Statamic routes are turned off

**Reply by CodeMooseVTG** [reply]
Didn't find an explanation, but got a workaround for now

**Comment by CodeMooseVTG** [comment]
Still not 100% sure why this happened or why this fixes it - but moving the search route from `web.php` to `api.php` allows the route to resolve. This works for my case because it's a data route anyway, but if anyone has any insight as to why web routes are still causing 404s...
